bundle/definition/schema_test.go

```diff
@@ -322,17 +322,15 @@ func TestConvertValue(t *testing.T) {
 	is.Error(err)
 
 	pd.Type = "array"
-	_, err = pd.ConvertValue("nope")
-	is.Error(err)
-
-	_, err = pd.ConvertValue("123")
-	is.Error(err)
-
-	_, err = pd.ConvertValue("true")
-	is.Error(err)
+	out, err = pd.ConvertValue(`["chocolate", "chip", "cookies"]`)
+	is.NoError(err)
+	is.Equal([]interface{}{"chocolate", "chip", "cookies"}, out)
 
-	_, err = pd.ConvertValue("123.5")
+	out, err = pd.ConvertValue(`["chocolate" "chip" "cookies"]`)
```

Comment:
Looks like you removed the other testcases, for "nope", "123", etc. Was that intentional?

Reply:
Oh, yes, it was... I did the same when adding support for `object`.  I'd be down to add 'em back for both, though.  WDYT?

Reply:
In other places we don't as exhaustively check negatives like that, but I don't see the harm in keeping them either? I'll leave it up to you.

Reply:
Are single values coerced into arrays according to the schema?

Reply:
Thanks @carolynvs and @radu-matei .  I've updated the cases for array and object to unmarshal into `[]interface{}` and `map[string]interface{}` respectively, as indeed, if just unmarshal-ing into `interface{}`, there  would be no error if attempting to use `"cookies"` as an array-type parameter.

Let me know how this is looking to y'all now.